src/main/java/org/jabref/logic/importer/fetcher/transformers/BiodiversityLibraryTransformer.java

```diff
@@ -0,0 +1,97 @@
+package org.jabref.logic.importer.fetcher.transformers;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.model.strings.StringUtil;
+
+public class BiodiversityLibraryTransformer extends YearRangeByFilteringQueryTransformer {
+
+    private static final List<String> STOP_WORDS = List.of("a", "and", "for", "or", "with");
+
+    // These have to be integrated into the IEEE query URL as these are just supported as query parameters
+    // Journal is wrapped in quotes by the transformer
```

Comment:
I think, this is a left-over from IEEE and can be removed?

```suggestion
    // These have to be integrated into the IEEE query URL as these are just supported as query parameters
    // Journal is wrapped in quotes by the transformer
```